components/TravelMethodButtons/TravelMethodButtons.jsx

```diff
@@ -0,0 +1,51 @@
+import { Flex, Box, SimpleGrid } from "@chakra-ui/react";
+import { modesOfTransport } from "../../utils/constants";
+import { TravelMethodButton } from "./TravelMethodButton";
+
+export function TravelMethodButtons({ handleTransPortMode }) {
+  return (
+    <Flex width={"fit container "}>
+      <SimpleGrid
+        columns={3}
+        id="selector"
+        width={["305px", "548px"]}
+        height={["345px", "304px"]}
+        mr={"auto"}
+        ml={"auto"}
+        mt={"6px"}
+      >
+        {modesOfTransport.map((mode, i) => (
+          <Box height="80px" textAlign={"center"} key={mode}>
+            {/* buttons */}
+            <TravelMethodButton
+              handleTransPortMode={handleTransPortMode}
+              mode={mode}
+              ind={i}
+            />
+            {/* <Button
+              fontSize={["16px", "16px"]}
+              color="#044B7F"
+              height={["100px", "80px"]}
+              width={["91.67px", "150px"]}
+              border="1px"
+              onChange={handleTransportMode}
+              colorScheme="#044B7F"
+              value={mode}
+            >
+              <Box
+                pos="absolute"
+                width={["30px", "32px"]}
+                height={["30px", "32px"]}
+                fontSize={"16px"}
+                mb={"22px"}
+              >
+                <Icon as={transportIcon[i]} />
+              </Box>
+              {mode}
+            </Button> */}
```

Comment:
Please remove the commented out code
```suggestion
```

Reply:
This request has been addressed in the latest commit.